packages/helpermodules/abstract_plans.py

```diff
@@ -33,10 +33,21 @@ class ScheduledLimit:
     soc_scheduled: int = 80
 
 
+@dataclass
+class TimeLimit:
```

Comment:
Die Klasse enthält die selben Attribute wie [https://github.com/openWB/core/blob/05b5b7312186f78d0d197aff282b32f478343c99/packages/control/ev.py#L62](url). Für die Limits reicht eine gemeinsame Klasse.

Reply:
Konkret hatte ich Probleme mit dem Import der Limit Klasse in abstract_plans, gab wohl einen zyklischen Veweis. Mit den Anpassungen fürs Zielladen wurde die Klasse von abstract_plans nach ev verschoben, wenn ich das wieder rückgängig mache könnte das wieder klappen.
https://github.com/openWB/core/commit/a2f72e3c2287837103fd7a33eb46cc04d769b4ff#diff-2fbc3606087f5749444f7c3f9ac1e33680b06813a783c67b99be8a2b1c8cf56cL15

Reply:
okay, das sollte nun passen.